fuzz/.gitignore

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Is it possible to remove the newline here?

Reply:
Done